README.md

```diff
@@ -1,5 +1,9 @@
 # Tomato: static blog generator
 
+[![Build Status](https://travis-ci.org/ribacq/tomato.svg?branch=master)](https://travis-ci.org/ribacq/tomato)
+[![Coverage Status](https://coveralls.io/repos/github/ribacq/tomato/badge.svg?branch=master)](https://coveralls.io/github/ribacq/tomato?branch=master)
+[![GoDoc](https://godoc.org/github.com/ribacq/tomato?status.svg)](https://godoc.org/github.com/ribacq/tomato)
+x
```

Comment:
This problem is fixed in the next commit (f573f097453e3190f23f468257971952b9a7474a).